doc/controls/ShadowContainer.md

```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
 BlurRadius | double | The radius of the blur that will be applied to the shadow **[0..100]**.
 Spread | double | The spread will inflate or deflate (if negative) the control shadow size **before** applying the blur.
 
+As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See below an image that illustrate its usage:
+
+![CSS box-shadow without inset](../assets/box-shadow.png)
```

Comment:
It looks like this and the next images are the wrong way around.
The text immediately above talks about IsInner/inset.